.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: false
+language: java
+jdk:
+- oraclejdk8
+before_cache:
+- rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+- rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+- rm -fr $HOME/.gradle/caches/*/fileHashes/
+- rm -fr $HOME/.gradle/caches/jars-3/*/buildSrc.jar
+cache:
+  directories:
+  - "$HOME/.gradle/caches/"
+  - "$HOME/.gradle/wrapper/"
+deploy:
+  provider: releases
+  api_key:
+    secure: iP2O3E7KrVlnw6E0acQ1iz2JEAwzAtMDUXbls14OlrZOz3Sc1Ti1UlF87F8FWaxpTGd2DnMue06XyO4XyTV4ijohWskxDL0BVOgKjqPtjLLB5rIxpVEW7BVM3jUc0gn7xAz2Ds2GI5gqDXBo7o5dGTB34KtaQYNjVejNOt3XfkKvcNSw+cHL9LU/lq0yAFmrKPkxDeUWmm8QmYzuXoGW/KiRcuNWe29Lg0bDoZ2j3DeA5qi8OyeZCa6I9t5rBhzDZa3RsA2XzsuxYGFHNL+jmZbQA62XAq6iP+g+JdpDojgWbzN7E/kbOS1MQcRoKX+3IyHKoOJbAS/nO439kg7s3qagJV+VkWRt2VrgkzWWn1vTUzE0u+NfadPhjtNiy0emNsqowGFdOiDVko0/OckQHDR1s9sxPt2akXcQJ82wPzg78ULdh+hJb/mr8SHcuxD5enV2mxRvxsRBMEJsluMOwg5q5k+EJkA0JOy5ZFF82z2+bA3nlma+Jcx6ymJBqez8SooYEekG7wR2lBHaQ17TamX5SI86k1yGK6hOSiHwmT8Fq50ZRF9kGcz2VR3Rp9WjcA6SN3Vs/t3pBxQUe/HDXzhW0U+aCxNLXmN+XG8IKdeNev0E5UDg1c1P2KVxap+gPxfQetBLuOfPl0KPGZ++bm/ZBkaQ5W2wxCwrUs8uuLQ=
```

Comment:
did you check these credentials into version control on purpose?

Reply:
These are the encrypted credentials...